hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/x509/SecurityConfig.java

```diff
@@ -216,6 +223,16 @@ public Duration getDefaultCertDuration() {
     return defaultCertDuration;
   }
 
+  /**
+   * Returns the number of days within which before expiration a certificate
```

Comment:
Sorry for missing this, but for consistency's sake could you also change the description here, how it was done in HddsConfigKeys?

Reply:
yes, you are right, done